libr/bin/p/bin_art.c

```diff
@@ -130,8 +131,17 @@ static RBinInfo *info(RBinFile *bf) {
 	return ret;
 }
 
+static bool check_buffer(RBuffer *buf) {
+	char tmp[4];
+	int r = r_buf_read_at (buf, 0, (ut8 *)tmp, sizeof (tmp));
+	return r == 4 && !strncmp (tmp, "art\n", 4);
+}
+
 static bool check_bytes(const ut8 *buf, ut64 length) {
```

Comment:
this must be removed

Reply:
not now, because the code still uses check_bytes. I plan to remove check_bytes in a future PR, but for now I'm making sure check_bytes rely on check_buffer.